README.md

```diff
@@ -0,0 +1,136 @@
+HighLine
+========
+
+by James Edward Gray II
```

Comment:
I didn't know about the Ruby Quiz, great to know!

I'm trying to look at some projects to see if there's a standard but couldn't find it. Everybody does differently. 

* Devise has a ```by Plataformatec``` the same position as ours and points to the GitHub repo contributors page.
* Locomotive has a ```Contact``` section
* Minitest has a ```Copyright © Ryan Davis, seattle.rb``` at the LICENSE
* RSpec completely misses any indication of whom is the original author or the currrent maintainers.
* Rails has a separate page with contributors by commit count. But there's no easy way to guess DHH is the author.
* Discourse has a separate ```docs/AUTHORS.md``` file that are very descriptive of the competences of each core contributor. https://github.com/discourse/discourse/blob/master/docs/AUTHORS.md

I like Discourse approach. What do you think @JEG2 ?

It would be something like:

# The Core HighLine Team

* [James Edward Gray II](https://github.com/JEG2) - Author
* [Gregory Brown](https://github.com/practicingruby) - Core contributor
* [Abinoam P. Marques Jr.](https://github.com/abinoam) - Core contributor

*For a list of people who have contributed to the codebase, see [GitHub's list of contributors](https://github.com/JEG2/highline/contributors).*

As our is shorter than Discourse, I think we could let it altogether at the README.md (no separate AUTHORS.md file, yet).

By the way, I have been in conversations with @matugm at the codenewbie slack. I like his code, suggestions and ideas. It would be good if he is interested in join in the core contributors team.